pkg/kubelet/kubelet.go

```diff
@@ -2047,19 +2047,36 @@ func (kl *Kubelet) SyncTerminatingPod(_ context.Context, pod *v1.Pod, podStatus
 
 	kl.probeManager.StopLivenessAndStartup(pod)
```

Comment:
@matthyx 

About what you mentioned in slack, the kubelet calls StopLivenessAndStartUp here before executing the prestop hook. Doesn't it?